python-lists-and-tuples/solution.py

```diff
@@ -0,0 +1,22 @@
+L = 'abcdefghijklmnopqrstuvwxyz'
+
+def create_combine_two(input):
+  # Create a list of numbers corresponding to the letters
+    L1 = []
+    for letter in range(len(input)):

```

Comment:
This is technically correct but it would probably be better to use a different variable name than "letter" since this is for the list of indices.